CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# [5.11.0] - 2024-04-23
```

Comment:
`[5.11.0]: https://github.com/WeInfuse/change_health/compare/v5.10.0...v5.11.0`

Should be added towards the bottom of this file

Reply:
Done, thanks!